### DISCUSSION TITLE
domino/14.0.0/admin/index

### DISCUSSION THREAD

**Discussion by giscus** [HCL-TECH-SOFTWARE/HCLSoftware-Documentation-Home#79] [Q&A]
# domino/14.0.0/admin/index

https://help.hcl-software.com/domino/14.0.0/admin/index.html

**Comment by KathyRatty** [comment]
HI ANYONE

**Reply by gthiel-id** [reply]
Hello, to help the writer improve the content of the documentation at that url, please share your observation or concern at the bottom of the specific page or section that prompted you to make a comment, giving details that allow us to address it.
